aiven/client/__init__.py

```diff
@@ -4,3 +4,5 @@
 # See the file `LICENSE` for details.
 
 from .client import AivenClient  # noqa
```

Comment:
Now the ignore can be removed ("works on my PC"):
```suggestion
from .client import AivenClient
```